src/main/java/com/meilisearch/sdk/model/TaskInfo.java

```diff
@@ -15,12 +14,7 @@ public class TaskInfo {
     protected int taskUid = 0;
     protected String indexUid = "";
     protected String type = null;
-    protected String duration = "";
```

Comment:
Were those attributes moved to "Task"/"TaskDetails"?

Reply:
They have not been moved, they are still in `Task` but from the v0.30.0 there are only these 4 fixed fields left `taskUid`, `indexUid`, `type`, `enqueuedAt`.

Reply:
Nice, cool!